src/index.html

```diff
@@ -15,27 +15,24 @@ <h1>Leaderboard</h1>
         <section>
           <h2>Recent scores</h2>
 
-          <button type="button">Refresh</button>
+          <button type="button" class="refresh-btn">Refresh</button>
         </section>
         
         <div class="recent-scores-container">
-          <div class="recent-score"><p>Name: 100</p></div>
-          <div class="recent-score"><p>Name: 200</p></div>
-          <div class="recent-score"><p>Name: 300</p></div>
-          <div class="recent-score"><p>Name: 400</p></div>
-          <div class="recent-score"><p>Name: 500</p></div>
         </div>
       </div>
 
       <section class="section-2">
         <h2>Add your score</h2>
 
         <form class="form">
-          <input type="text" class="user-name-field" value="" name="user-name" placeholder="Your name"/><br><br>
-          <input type="text" class="user-score-field" value="" name="user-score" placeholder="Your score"/><br><br>
-          <input type="submit" value="Submit"/>
+          <input type="text" class="user-name-field" name="user-name" placeholder="Your name"/><br><br>
+          <input type="text" class="user-score-field" name="user-score" placeholder="Your score"/><br><br>
+          <input type="submit" class="submit-btn" value="Submit"/>
         </form>
       </section>
     </main>
+
+    <script type="module" src="./index.js"></script>
   </body>

```

Comment:
- [ ] [OPTIONAL] When a user sends a long score number, the page design can break. kindly consider adding some kind of form validation (max-width) so the user can't submit a big number as a score

![Screenshot_20220831_202741](https://user-images.githubusercontent.com/25227022/187766273-2f6c98c3-544e-40d1-9304-e99790e9f393.png)